### DISCUSSION TITLE
How do you get the covid19 vaccination donation data on the owid website?

### DISCUSSION THREAD

**Discussion by mohamed-tayeh** [owid/covid-19-data#2526] [General]
I saw that [this](https://data.covid19taskforce.com/data/tables) is the data source

I am not sure how to dynamically retrieve the data since I saw that it is not in this repo as well and I was interested in it

**Comment by edomt** [comment]
Hi @mohamed-tayeh 

Indeed the data comes from [this dashboard](https://data.covid19taskforce.com/data/tables). I'm not aware of any easy way to fetch it dynamically, although using something like [Selenium](https://www.selenium.dev/) would probably work (to load the page and scrape the table). The data is only updated about once a month, and it's still fairly small, so I've updated the data manually so far.

**Reply by mohamed-tayeh** [reply]
Thank you!
